lib/matplotlib/ticker.py

```diff
@@ -253,6 +253,9 @@ def __call__(self, x, pos=None):
         """
         raise NotImplementedError('Derived must override')
 
+    def format_ticks(self, values):
```

Comment:
I think that this needs an optional `positions` argument.  If not `None`, it would have to have a length matching `values`).  Otherwise, this method won't work for things like `FixedFormatter` and `FuncFormatter`.

Reply:
(note to self: or at least make sure it doesn't break the above-mentioned formatters)

Reply:
Actually they just work fine with this patch, as can be checked with the ticks_and_spines/tick-formatters.py example.

Reply:
Now I see.  I was confused.